core/domain/stats_domain.py

```diff
@@ -145,6 +145,18 @@ def create_default(cls, exp_id, exp_version, state_stats_mapping):
         """
         return cls(exp_id, exp_version, 0, 0, 0, 0, 0, 0, state_stats_mapping)
 
+    def compute_answer_count(self):
+        """Compute the answer count for the exploration stats."""
```

Comment:
Should have a Returns: docstring, since this returns something. (Ditto for the method above, incidentally.)

Reply:
Done.